fhir-tools/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:lts
```

Comment:
good, could you please also add dockerfile for the au-fhir-tools? I guess it should be the same.